templates/zerver/home.html

```diff
@@ -42,6 +42,11 @@
 
         <p>{% trans 'Want to' %} <a href="#" class="empty_feed_join">{% trans 'join it' %}</a>?</p>
       </div>
+      <div id="non_existing_stream_message" class="empty_feed_notice">
+        <h4>{% trans "This stream does not exist. Manage your streams and subscriptions on " %}
```

Comment:
Instead of `trans` can you please use `_()` function? `upstream:master` now uses Jinja2 and `_()` works in both Django and Jinja2. Thanks.